.buildkite/pipeline.yml

```diff
@@ -107,7 +107,7 @@ steps:
   #
   - group: ":bitbar: E2E Tests"
```

Comment:
An existing problem, but maybe we can remove the mention of bitbar here as it doesn't apply to everything in the group?